shaders/EditorHexGrid.shader

```diff
@@ -1,27 +1,48 @@
 shader_type spatial;
 render_mode unshaded;
 
-// TODO: implement our own version of procedural hex grid tiling for a better
+// Implemented our own version of procedural hex grid tiling for a better
 // looking and more uniform hexes.
 // See andrewhungblog.wordpress.com/2018/07/28/shader-art-tutorial-hexagonal-grids/
 
-uniform sampler2D gridTexture;
 uniform sampler2D maskTexture;
 uniform vec4 color : hint_color;
+uniform float lineWidth = 0.02;
+uniform float edgeLength = 1.3;
 
-uniform vec2 gridSize;
+const vec2 hexSize = vec2(1.7320508, 1); // 1.7320508 = sqrt(3)
 
 varying vec3 worldPos;
+varying float minLineWidth;
+
+// Returns normalized distance to the nearest hex center.
+float calcHexCenterDistance(vec2 coord)
+{
+    vec4 hexCenter = round(vec4(coord, coord - vec2(1., .5)) / hexSize.xyxy);
```

Comment:
I really dislike writing numbers out as `1.` or `.5` I would greatly appreciate if you could change all these to include the leading / trailing zero `1.0, 0.5`